distribution/src/assembly/source-assembly.xml

```diff
@@ -47,6 +47,7 @@
                 <exclude>.gitignore</exclude>
                 <exclude>.dockerignore</exclude>
                 <exclude>.travis.yml</exclude>
+                <exclude>README.md</exclude>
```

Comment:
Should we exclude this?

Reply:
I think so, the idea was that `README.md` is for github, and `README` is a plaintext with a bit more info that is packed into the source and binary release packaging to make that a bit more friendly.

That said, if you think there is a good reason to have both `README` and `README.md` in the source archive, I think that would probably be fine and I can add it back.

Reply:
Hmm. I mean, is it better to add both, so that ppl can still choose whatever they prefer? The markdown file is not best for terminal users, but some terminal tools still support markdown syntax. For example, even `vi` shows comments, texts, and titles with different colors. It looks easier to read to me.

Reply:
I think it's fine to leave out `README.md` for the source/binary releases, I think it's a bit confusing to have two copies in the release assemblies

Reply:
Ok, I don't have strong opinion about this.